src/tsickle.ts

```diff
@@ -127,6 +128,9 @@ const VISIBILITY_FLAGS: ts.ModifierFlags =
  * Another Rewriter subclass manages .ts => externs translation.
  */
 class ClosureRewriter extends Rewriter {
+  /** A mapping of aliases for symbols in the current file, used when emitting types.  */
```

Comment:
Can you elaborate here on what "aliases for symbols" are?  As written this could be a TypeScript type = foo alias, a Closure typedef, or some other thing.